app/controllers/JourneyRecoveryController.scala

```diff
@@ -18,37 +18,52 @@ package controllers
 
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.binders.RedirectUrl._
-import views.html.{JourneyRecoveryContinueView, JourneyRecoveryStartAgainView}
+import config.FrontendAppConfig
 import uk.gov.hmrc.play.bootstrap.binders._
+import config.Constants.SRN
 import controllers.actions.IdentifierAction
 import play.api.Logging
+import views.html.{JourneyRecoveryContinueView, JourneyRecoveryStartAgainView}
+import models.SchemeId.Srn
 import play.api.i18n.I18nSupport
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
 
 import javax.inject.Inject
 
 class JourneyRecoveryController @Inject()(
   val controllerComponents: MessagesControllerComponents,
+  config: FrontendAppConfig,
   identify: IdentifierAction,
   continueView: JourneyRecoveryContinueView,
   startAgainView: JourneyRecoveryStartAgainView
 ) extends FrontendBaseController
     with I18nSupport
     with Logging {
 
-  def onPageLoad(continueUrl: Option[RedirectUrl] = None): Action[AnyContent] = identify { implicit request =>
-    val safeUrl: Option[String] = continueUrl.flatMap { unsafeUrl =>
-      unsafeUrl.getEither(OnlyRelative) match {
-        case Right(safeUrl) =>
-          Some(safeUrl.url)
-        case Left(message) =>
-          logger.info(message)
-          None
+  def onPageLoad(continueUrl: Option[RedirectUrl] = None, answersSavedDisplayVersion: Int = 0): Action[AnyContent] =
```

Comment:
I'm not super clear on what `answersSavedDisplayVersion` does

Reply:
Which of these two to render:
```
journeyRecovery.guidance2.version1 = We have saved your answers.
journeyRecovery.guidance2.version2 = We have saved your answers in this section until you are signed out.
```
I tried to do it with enum, but it got complicated so I went with Int up for now.